content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -36,7 +36,7 @@ Datadog instrumentation to avoid duplicate spans in the trace.
 | Java     | v1.35.0                  |
 | Python   | v2.10.0                  |
 | Ruby     | v2.1.0                   |
-| Go       | TBA                      |
+| Go       | v1.65.0 (Span event support planned for v1.67.0)|
```

Comment:
@neko-dd , any thoughts on this formatting/wording?

Reply:
We do not document the future, so this doesn't work. It should just have the current version, and get another update when the 1.67 version is available.

Reply:
If you need to say that span events are not supported somewhere, you can add that, but not in a way that implies that they will be in the future.